src/test/java/org/mockitousage/stubbing/StrictStubbingEndToEndTest.java

```diff
@@ -93,6 +99,25 @@ public void run() {
         }
     }
 
+    public static class UnnecessaryStubbingWithImplicitEqMatchers {
```

Comment:
The name does not seem correct - we don't use explicit 'eq' matchers in those tests.

Reply:
Done.